build-llvm11.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: LLVM 11 (recommended)
+subtitle: Building KLEE with LLVM 11
+slug: getting-started
+---
+
+{% include version_warning.md %}
+
+The current procedure for building KLEE with LLVM 11 (recommended) is outlined below.
+
+
+**NOTE:** KLEE is currently tested on Linux x86-64 (particularly Ubuntu), FreeBSD and macOS.
+There is no support for uClibc and the POSIX environment under macOS.
+KLEE does not work under x86-32.
+
+1. **Install dependencies:** KLEE requires all the dependencies of LLVM (see [here](http://llvm.org/docs/GettingStarted.html#requirements)), and some more. In particular, you should install the programs and libraries listed below.
+
+   Under Ubuntu, use:
+   ```bash
+   $ sudo apt-get install build-essential cmake curl g++-multilib gcc-multilib git libcap-dev libgoogle-perftools-dev libncurses5-dev libsqlite3-dev libtcmalloc-minimal4 python3-pip unzip doxygen
+   ```
+
+   Under macOS, run:
+   ```bash
+   $ brew install curl git cmake python unzip gperftools sqlite3 doxygen bash
+   ```
+
+   You should also install `lit` to enable testing, `tabulate` to support additional features in `klee-stats` and `wllvm` to make it easier to compile programs to LLVM bitcode:
+
+   ```bash
+   $ sudo pip3 install lit wllvm
+   $ sudo apt-get install python3-tabulate
+   ```
+
+   Use `--user` for `pip3` to install packages for the current user only and make sure that `~/.local/bin` is in your `PATH`.
```

Comment:
`--user` without `sudo`

And the proper check is:
```
base_path=$(python3 -m site --user-base)
export PATH="${base_path}/bin:$PATH"
```
As it's different under macOS